src/main/routes/case-data-probate.ts

```diff
@@ -13,10 +13,10 @@ var env = nunjucks.configure({ autoescape: true });
 env.addFilter('date', dateFilter);
 env.addFilter('money', numeralFilter);
 
-router.get("/jurisdictions/:jid/case-types/:ctid/cases/:cid/probate", (req, res, next) => {
-  getCase(req, req.params.jid, req.params.ctid, req.params.cid)
+router.get("/jurisdictions/PROBATE/case-types/GrantOfRepresentation/cases/:cid", (req, res, next) => {
```

Comment:
Not sure if Express will match this route over the standard pattern `"/jurisdictions/:jid/case-types/:ctid/cases/:cid"` in `case-data.ts` - it would depend on which route was declared first, as `"/jurisdictions/PROBATE/case-types/GrantOfRepresentation/cases/:cid"` could be matched by the route in `case-data.ts`. @vlaurin?

Reply:
That was my only worry
It did seem to work for my PROBATE services
I think everything else would go through case-data.ts as normal
Wait for @vlaurin to review...

Reply:
It all boils down to the order in which the routes are registered in the application.

But I think that's bound to lead to confusion, if not errors, as it seems fragile to rely on that.
Could you please prefix the whole path with something Probate-specific, like `/probate/jurisdictions/PROBATE/case-types/GrantOfRepresentation/cases/:cid`?

Reply:
I did not think I could do that because this path is defined in ccd-data-store-api as the default print url so it will be used by all users of the print service

Reply:
Yes, I'm suggesting you don't use the print callback from `ccd-data-store-api`, but your own print callback.

Reply:
so we would have to create the list of doc urls for ourself?

Reply:
It would be better, yes. That way you can create the link in the format that you want and we are sure it's not going to clash with CCD's default one. Also, this will allow you to have different links for different types of users, as you mentioned 2 different templates I think.

Reply:
If I try this then I need to do my own auth too?

Reply:
It's a regular callback, same as the ones you are already using. It's not user-facing. You can (and probably should) put it alongside your existing callbacks implementations.